modules/artifact/src/test/java/org/mule/runtime/module/artifact/api/classloader/FilteringArtifactClassLoaderTestCase.java

```diff
@@ -133,6 +137,61 @@ public void loadsExportedResource() throws ClassNotFoundException, IOException {
     assertThat(resource, equalTo(expectedResource));
   }
 
+  @Test
+  public void getPackageExported() throws ClassNotFoundException, IOException {
```

Comment:
Rename test methods, should be getExportedPackage/getNoExportedPackge/getExportedPackages